difflint/lint.py

```diff
@@ -0,0 +1,62 @@
+import os.path
```

Comment:
Also, raises an interesting issue; any new code that @BlackVegetable adds shouldn't be copyright Endless Mobile, Inc. I think the usual solution in open source projects is for each contributor to add a copyright line to the header, reflecting their (personal or employer's, as the case may be) copyright.